docs/zkapps/tutorials/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: zkApp Developer Tutorials
+hide_title: true
+sidebar_label: About Tutorials
+description: zkApp developer tutorials provide a hands-on walk-through of use cases that teach you how to achieve a defined goal.  
+keywords:
+  - quick start
+  - zkapps
+  - zero knowledge proof programming
+  - zk proof
+  - zk
+  - snarkyjs
+  - blockchain
+  - mina
+  - typescript
+---
+
+# zkApp Developer Tutorials
+
+zkApp developer tutorials are a hands-on walk-through of use cases that teach you how to achieve a defined goal.
+
+To meet other developers building zkApps with SnarkyJS, participate in the [#zkapps-developers](https://discord.com/login?redirect_to=%2Fchannels%2F484437221055922177%2F915745847692636181) channel on Mina Protocol Discord.
+
+## Prerequisites
+
+Each tutorial has been tested with specific versions:
+
+- [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli)
+- [SnarkyJS](https://www.npmjs.com/package/snarkyjs)
+
+    SnarkyJS is automatically included when you create a project using the Mina zkApp CLI.
+
+- Other dependencies as noted
+
+### Install the Mina zkApp CLI
+
+To install the Mina zkApp CLI:
+
+```sh
+$ npm install -g zkapp-cli
+```
+
+To confirm successful installation:
+
+```sh
+$ zk --version
+```
+
+## Dependencies
+
+To use the zkApp CLI and SnarkyJS, your environment requires:
+
+- NodeJS v16 and later (or NodeJS v14 using `--experimental-wasm-threads`)
+- NPM v6 and later
+- Git v2 and later
+
+Use a package manager to install the required versions and upgrade older versions if needed. Package managers for the supported environments are:
+
+- MacOS [Homebrew](https://brew.sh/)
+- Windows [Chocolatey](https://chocolatey.org/)
+- Linux
+
+  - apt, yum, and others
+
+  On Linux, you might need to install a recent Node.js version by using NodeSource. Use [deb](https://github.com/nodesource/distributions#debinstall) or [rpm](https://github.com/nodesource/distributions#rpminstall) as recommended by the Node.js project.
+
+To verify your installed versions, use `npm -v`, `node -v`, and `git -v`.
+
+## Tips
+
+The full source code for tutorials is provided in  the [examples/zkapps/](https://github.com/o1-labs/docs2/tree/main/examples/zkapps/) directory on GitHub. While you're there, give the `/docs2` repository a star so that other zk developers can learn to build a zkApp! 
```

Comment:
Unrelated to this pr question. Did we ever discuss just having a docs repo instead of docs2?

Reply:
@ymekuria  I wondered the same thing! 
Web search for mina docs always puts the legacy mina repo at the top of search results

<img width="629" alt="image" src="https://github.com/o1-labs/docs2/assets/10067215/f43bf150-7c19-4e3a-933e-bdf310a630b3">

Now that the https://github.com/MinaProtocol/docs/ is archived, it is read only 
Would we be able to unarchive and update this statement in https://github.com/MinaProtocol/docs/blob/main/README.md ?

> Note: This repo contains the old docs website. The new docs website is located at https://github.com/o1-labs/docs2

Reply:
@barriebyron I think we should bring this up again with the team during open forum or sometime like that

Reply:
Let's talk about it in our next team meeting https://o1-labs.slack.com/archives/C028Q27R8UC/p1686942417584859